client/cmd/dexc/config.go

```diff
@@ -75,6 +75,7 @@ func defaultHostByNetwork(network dex.Network) string {
 type Config struct {
 	AppData      string `long:"appdata" description:"Path to application directory."`
 	Config       string `long:"config" description:"Path to an INI configuration file."`
+	SiteDir      string `long:"sitedir" description:"Path to the 'site' directory with packaged web files. Unspecifed = default is good in most cases."`
```

Comment:
Would you add something to the sample config https://github.com/decred/dcrdex/blob/master/client/cmd/dexc/sample-dexc.conf?